src/Applications/LDAS_App/ldas_setup

```diff
@@ -1241,8 +1241,8 @@ class LDASsetup:
                     elif 'MY_NODES' in line :
                         line_ = line.replace('MY_NODES',str(self.optRmInp['nodes']))
                         fout.write(line_.replace('MY_NTASKS_PER_NODE',str(self.rqdRmInp['ntasks-per-node'])))
-                        if int(self.rqdRmInp['ntasks-per-node']) > 28:
-                           fout.write("#SBATCH --constraint=sky\n")
+                        if int(self.rqdRmInp['ntasks-per-node']) > 40:
+                           fout.write("#SBATCH --constraint=cas\n")
```

Comment:
if tasks-per-node is between 28 and 40,  the constrain should be sky. Without constrain, it goes to haswell and may cause error.

Reply:
```suggestion
                      if (28 < int(self.rqdRmInp['ntasks-per-node']) and int(self.rqdRmInp['ntasks-per-node']) <= 40) :
                         fout.write("#SBATCH --constraint=sky\n")
```

Reply:
```suggestion
                           fout.write("#SBATCH --constraint=cas\n")
                        if (28 < int(self.rqdRmInp['ntasks-per-node']) and int(self.rqdRmInp['ntasks-per-node']) <= 40) :
                           fout.write("#SBATCH --constraint=sky\n")```